core/templates/dev/head/css/oppia.css

```diff
@@ -3901,6 +4214,46 @@ md-card.preview-conversation-skin-supplemental-card {
   right: 0;
 }
 
+/*
+  Styles for Setting tab
```

Comment:
Setting --> Settings

But if it's only the settings tab, shouldn't we put it in that page instead?

Reply:
This is also used for settings which come in modals, I've moved advanced feature CSS to settings tab and containers are now here